src/test/ui/parser/raw/raw-str-long.stderr

```diff
@@ -0,0 +1,12 @@
+error: unterminated raw string
+  --> $DIR/raw-str-long.rs:2:13
+   |
+LL |     let a = r##"This
+   |             ^^^ unterminated raw string
+help: you might have meant to end the raw string here
+   |
+LL |     '##;
+   |     ^^^
```

Comment:
I'm worried about the `'##` not being `"##`.

Reply:
True, the reason for it is, that I look for `#`, not for `"#[...]`. I think I can change this.